src/main/java/controller/GameController.java

```diff
@@ -112,13 +112,6 @@ public boolean playerRelativeWalk()
         System.out.println("Player " + data.currentPlayer.getUserId() + " moved to " + data.currentPlayer.getPosition());
         return flag;
     }
-    public void playerAbsoluteWalk(int BlockId)
```

Comment:
Do we not need the `playerAbsoluteWalk`?

Did we scrap the idea of having destiny cards to move you to a specific place?

Reply:
Originally when writing the `playerRelativeWalk` method, we just wrote this method together, but later we found it to be useless so I deleted it.